Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>*.iml\r\n.gradle\r\n/local.properties\r\n/.idea/caches\r\n/.idea/libraries\r\n/.idea/modules.xml\r\n/.idea/workspace.xml\r\n/.idea/navEditor.xml\r\n/.idea/assetWizardSettings.xml\r\n.DS_Store\r\n/build\r\n/captures\r\n.externalNativeBuild\r\n.cxx\r\nlocal.properties\r\nconfig.properties\r\n*.log\r\n./gradle/libs.versions.toml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 05cf07318c597501f7a4e55a01ba8e106a2b7038)
+++ b/.gitignore	(date 1727878495410)
@@ -15,4 +15,4 @@
 local.properties
 config.properties
 *.log
-./gradle/libs.versions.toml
\ No newline at end of file
+gradle/libs.versions.toml
